src/pages/GetInvolved/GetInvolved.js

```diff
@@ -1,10 +1,45 @@
-import React from 'react';
-import Layout from 'components/Layout/Layout';
+import React from "react";
+import { useSelector } from "react-redux";
+import Layout from "components/Layout/Layout";
+import { Link } from "react-router-dom";
```

Comment:
lets switch the order of these
```suggestion
import { Link } from "react-router-dom";
import Layout from "components/Layout/Layout";
```